WebApi/Controllers/CustomersController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using FluentValidation.Results;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Net;
+using WebApi.Models;
+using WebApi.Repository;
+
+namespace WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomersController : Controller
+    {
+        private ICustomersRepository _repository;
```

Comment:
Como não estamos mudando valor desse field devemos definir como readonly
```suggestion
        private readonly ICustomersRepository _repository;
```